libs/cardano-ledger-core/src/Cardano/Ledger/PoolDistr.hs

```diff
@@ -48,16 +54,21 @@ import NoThunks.Class (NoThunks (..))
 -- delegated to a registered stake pool.
 data IndividualPoolStake c = IndividualPoolStake
   { individualPoolStake :: !Rational
```

Comment:
I have updated the comment to reflect that proposal-deposits are not considered in this ratio but are added to the absolute fields.

> This is a ratio of (individualTotalPoolStake - proposal deposits) to (pdTotalActiveStake - proposal deposits)

Reply:
No, please don't. We do not need to mention this, because this information about proposal deposits is only relevant for Conway and only for voting, so this will only be confusing for anyone who only cares about stake distribution with respect to leader election